oap-server/server-receiver-plugin/configuration-discovery-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/recevier/configuration/discovery/AgentConfigurationsWatcher.java

```diff
@@ -73,7 +73,7 @@ public String value() {
     public AgentConfigurations getAgentConfigurations(String service) {
         AgentConfigurations agentConfigurations = agentConfigurationsTable.getAgentConfigurationsCache().get(service);
         if (null == agentConfigurations) {
-            return emptyAgentConfigurations;
+            return agentConfigurationsTable.getAgentConfigurationsCache().getOrDefault("default-config", emptyAgentConfigurations);
```

Comment:
This logic should work directly, isn't it? As from YAML perspective, `default-config` is a `service name` only.

Reply:
Please correct me I missed anything.